examples/stress-test/run.py

```diff
@@ -0,0 +1,202 @@
+# Apache Software License 2.0
+#
+# Copyright (c) ZenML GmbH 2025. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import time
+from typing import Any, Dict
+
+import click
+
+from zenml import get_step_context, pipeline, step
+from zenml.client import Client
+from zenml.config import DockerSettings
+from zenml.config.docker_settings import PythonPackageInstaller
+from zenml.integrations.kubernetes.flavors.kubernetes_orchestrator_flavor import (
+    KubernetesOrchestratorSettings,
+)
+from zenml.integrations.kubernetes.pod_settings import KubernetesPodSettings
+
+kubernetes_settings = KubernetesOrchestratorSettings(
+    pod_startup_timeout=600,
+    pod_settings=KubernetesPodSettings(
+        resources={
+            "requests": {"cpu": "100m", "memory": "300Mi"},
+            "limits": {"memory": "350Mi"},
+        },
+        node_selectors={"pool": "workloads"},
+        tolerations=[
+            {"key": "pool", "operator": "Equal", "value": "workloads", "effect": "NoSchedule"}
+        ],
+        env=[{"name": "ZENML_LOGGING_VERBOSITY", "value": "debug"}],
+    ),
+    orchestrator_pod_settings=KubernetesPodSettings(
+        resources={
+            "requests": {"cpu": "100m", "memory": "300Mi"},
+            "limits": {"memory": "350Mi"},
+        },
+        node_selectors={"pool": "workloads"},
+        tolerations=[
+            {"key": "pool", "operator": "Equal", "value": "workloads", "effect": "NoSchedule"}
+        ],
+    ),
+)
+
+docker_settings = DockerSettings(
+    python_package_installer=PythonPackageInstaller.UV,
+    parent_image="public.ecr.aws/f7g3v0f1/zenml:stefan",
```

Comment:
Maybe leave this out so it uses the default?

Reply:
:+1: